.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -12,7 +12,7 @@ on:
       DEVELOPMENT_VERSION:
         required: true
         description: "Next development version"
-        default: "0.1.0-SNAPSHOT"
+        default: "8.7.0-SNAPSHOT"
```

Comment:
I don't think we should change the default here. The current default is a more sensible one